src/vector3d.f90

```diff
@@ -12,7 +12,8 @@ module m_slab
       class(field_t), pointer :: u1, u2, u3
       real :: xnu
    contains
-      procedure(field_op), deferred :: transport, div
+      procedure(field_op), deferred :: transport
+      procedure(field_op), deferred :: div
```

Comment:
That's to make nvfortran happy if I remember correctly?

Reply:
Yes, it doesn't compile when you list them in the same line